generators/readme/README.md

```diff
@@ -0,0 +1,18 @@
+# Readme generator
+
+A package to generate README files for the React component packages.
+
+## Prerequisites
+
+Each package should have the following fragment files in a `docs` folder:
+
+- `docs/usage-example.js`: a JS example for using the component.
+- `docs/additional-info.md`: any additional information to be rendered AFTER the generated sections.
```

Comment:
can you mention `docs/description.md` as well? also the fallback to `description` on `package.json` incase that file is missing?

Reply:
Ah yes, I forgot about that.